jiralib.el

```diff
@@ -424,6 +425,17 @@ will cache it."
           (jiralib-make-assoc-list (jiralib-call "getPriorities") 'id 'name)))
   jiralib-priority-codes-cache)
 
+(defvar jiralib-assignable-users-cache nil)
+(defun jiralib-get-assignable-users (key)
+  "Return an assoc list mapping user key and display name.
+
+This function will only ask JIRA for the list of name once, than
+will cache it."
+  (unless jiralib-assignable-users-cache
```

Comment:
Caching will not work well if the project changes, and a different project has a different list of assignable users.